hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/insights/insights.tsx

```diff
@@ -215,69 +264,107 @@ export class Insights extends React.Component<Record<string, object>, IInsightsS
+                                    allowSelectAll
+                                    isMulti
+                                    className='multi-select-container'
+                                    options={bucketOptions}
+                                    closeMenuOnSelect={false}
+                                    hideSelectedOptions={false}
+                                    value={selectedBuckets}
+                                    allOption={allBucketsOption}
+                                    isDisabled={isBucketSelectionDisabled}
+                                    onChange={this.handleBucketChange}
+                                  />
+                                </div>
+                              </Col>
+                            </Row>
+                          </Col>
+                        </Row>
+                        <Row>
+                          <Col>
+                            <Plot
+                              data={plotData}
+                              layout={
+                                {
+                                  width: 800,
+                                  height: 600,
+                                  title: 'File Size Distribution',
+                                  showlegend: false
+                                }
+                              } />
+                          </Col>
+                        </Row>
+                      </div> :
+                      <div>No data to visualize file size distribution. Add files to Ozone to see a visualization on file size distribution.</div>)}
+                </div>
+              }
+            </TabPane>
+            <TabPane key='2' tab={`Container Size`}>
+              {
+                <div className='content-div'>
+                  {isLoading ? <span><Icon type='loading'/> Loading...</span> :
+                    ((containerCountResponse && containerCountResponse.length > 0) ?
+                      <div>
+                        <Row>
+                          <Col>
+                            <Plot
+                              data={containerData}
+                              layout={
+                                {
+                                  width: 850,
+                                  height: 750,
+                                  font: {
+                                    family: 'Roboto, sans-serif',
+                                    size: 15
+                                  },
+                                  title: 'Container Size Distribution',
+                                  showlegend: true
+                                }
+                              }/>
+                          </Col>
+                        </Row>
+                      </div> :
+                      <div>No data to visualize Container size distribution. Add files to Ozone to see a visualization on Container size distribution.</div>)}
```

Comment:
```suggestion
                      <div>No data to visualize container size distribution. Add files to Ozone to see a visualization on container size distribution.</div>)}
```

Reply:
@dombizita 
Completed Change Suggested by you.